beets/library.py

```diff
@@ -701,27 +701,32 @@ def move_file(self, dest, operation=MoveOperation.MOVE):
         """
```

Comment:
It might be nice to mention in the docstring that this is "fail-stop," and to mention the kinds of exceptions that can be thrown.